### DISCUSSION TITLE
Assistance Required for Integrating Level 4 Autonomous Vehicle into CARLA Simulator

### DISCUSSION THREAD

**Discussion by mave8533** [carla-simulator/carla#7753] [General]
Dear CARLA Community,

I hope this message finds you well. I am currently working on a project to integrate a level 4 autonomous vehicle into the CARLA simulator. Despite my efforts, I have encountered several challenges and would greatly appreciate any guidance or assistance from those who have experience with similar projects.
I am particularly struggling with finding a suitable 3D model of a level 4 autonomous vehicle that can be easily integrated into the CARLA environment. Additionally, I am looking for detailed guidance on the steps required to achieve this integration, including any necessary code modifications or configurations.
If anyone has experience with integrating level 4 autonomous vehicles in CARLA or can point me to relevant resources, tutorials, or documentation, it would be incredibly helpful. Any advice on overcoming common challenges or best practices for this type of project would also be greatly appreciated.

Thank you very much for your time and assistance. I look forward to any suggestions or insights you can provide.

**Comment by ADASDevelopment** [comment]
#Let'sSupport
